redfish-core/lib/account_service.hpp

```diff
@@ -1163,6 +1184,30 @@ inline void getAcfProperties(
     }
     asyncResp->res.jsonValue["Oem"]["IBM"]["ACF"]["ACFInstalled"] =
         acfInstalled;
+
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    const std::variant<bool>& retval) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR << "D-Bus responses error: " << ec;
+            messages::internalError(asyncResp->res);
+            return;
+        }
+        const bool* allowed = std::get_if<bool>(&retval);
+        if (allowed == nullptr)
+        {
+            BMCWEB_LOG_ERROR << "Property 'allowed' is not bool";
+            messages::internalError(asyncResp->res);
+            return;
+        }
+        asyncResp->res.jsonValue["Oem"]["IBM"]["ACF"]["AllowUnauthACFUpload"] =
+            *allowed;
+        },
+        "xyz.openbmc_project.Settings",
+        "/xyz/openbmc_project/ibmacf/allow_unauth_upload",
+        "org.freedesktop.DBus.Properties", "Get",
```

Comment:
Would prefer this got moved to getProperty like the upstream code 
https://github.com/openbmc/bmcweb/blob/5e7e2dc585dc7a7e62d2b648b541e7f50c39ea5d/redfish-core/lib/led.hpp#L39

Reply:
Changed to use getProperty<bool>() like
```
    sdbusplus::asio::getProperty<bool>(
        *crow::connections::systemBus, "xyz.openbmc_project.Settings",
        "/xyz/openbmc_project/ibmacf/allow_unauth_upload",
        "xyz.openbmc_project.Object.Enable", "Enabled",
        [asyncResp](const boost::system::error_code& ec,
                    const bool allowUnauthACFUpload) {
...
        });
```